src/main/java/cabaletta/comms/Pipe.java

```diff
@@ -74,11 +74,11 @@ public iMessage receiveMessage() throws IOException {
 
         @Override
         public void close() {
-            closed = true; // Prevent further messages from being sent and received
+            Pipe.this.closed = true; // Prevent further messages from being sent and received
```

Comment:
this is cring

Reply:
It's easier to read, makes it more explicit that the closed flag resides in the parent class.

Reply:
this is based